src/Compponents/LogoutButton.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
+import {useDispatch, useSelector} from 'react-redux';
+import "./LogoutButton.css";
+import {setAuthorizedUser} from "../Store/slices/authorizedUserSlice";
+
+const LogoutButton: React.FC = () => {
+  const { logout, user: auth0User, isAuthenticated: auth0IsAuthenticated } = useAuth0();
+  const user = useSelector((state: any) => state.user);
+  const dispatch = useDispatch();
+
+  const username = auth0IsAuthenticated
+    ? auth0User?.nickname
+    : user?.user?.username;
+
+  const handleLogout = () => {
+    localStorage.removeItem('authToken');
```

Comment:
also create a separate file for the local storage management 
localStorage.ts and make all storage related operations there, import functions in the components